### ISSUE TITLE
5. step installation instruction fails on os x - "Disabling ICU" & "grpc: the connection is unavailable"

### ISSUE THREAD

**Issue by hkgrani** [dgraph-io/dgraph#449]
I tried to follow the five step process here : https://wiki.dgraph.io/Get_Started usng the simple install : "curl https://get.dgraph.io | bash"

"ls $ICU_DATA" give result : 
goldendata.rdf.gz	p
goldendata.schema	w

When trying to run "dgraph --schema goldendata.schema" I get 

```
2017/01/05 13:57:14 embedflag.go:20: Disabling ICU because we fail to create tokenizer: ICU new tokenizer error 2
2017/01/05 13:57:14 main.go:623: num_cpu: 4. prev_maxprocs: 4. Set max procs to num cpus
Starting commit routine.
2017/01/05 13:57:14 main.go:673: grpc server started.
2017/01/05 13:57:14 main.go:674: http server started.
2017/01/05 13:57:14 main.go:675: Server listening on port 8080
2017/01/05 13:57:14 worker.go:80: Worker listening at address: [::]:12345
Node with GroupID: 0, ID: 1
Node with GroupID: 1, ID: 1
Found hardstate: {Data:[] Metadata:{ConfState:{Nodes:[] XXX_unrecognized:[]} Index:0 Term:0 XXX_unrecognized:[]} XXX_unrecognized:[]}
Found hardstate: {Data:[] Metadata:{ConfState:{Nodes:[] XXX_unrecognized:[]} Index:0 Term:0 XXX_unrecognized:[]} XXX_unrecognized:[]}
Group 0 found 4 entries
Restarting node for group: 0
Group 1 found 4 entries
Restarting node for group: 1
raft2017/01/05 13:57:14 INFO: 1 became follower at term 2
raft2017/01/05 13:57:14 INFO: newRaft 1 [peers: [], term: 2, commit: 4, applied: 0, lastindex: 4, lastterm: 2]
raft2017/01/05 13:57:14 INFO: 1 became follower at term 2
raft2017/01/05 13:57:14 INFO: newRaft 1 [peers: [], term: 2, commit: 4, applied: 0, lastindex: 4, lastterm: 2]
{Term:2 Index:2 Type:EntryNormal Data:[] XXX_unrecognized:[]}
{Term:2 Index:2 Type:EntryNormal Data:[] XXX_unrecognized:[]}
2017/01/05 13:57:14 draft.go:371: group: 1 Addr: "localhost:12345" leader: true dead: false
----------------------------
2017/01/05 13:57:14 draft.go:371: group: 0 Addr: "localhost:12345" leader: true dead: false
----------------------------
====== APPLYING MEMBERSHIP UPDATE: {NodeId:1 Addr:localhost:12345 Leader:true RaftIdx:4}
----------------------------
Group: 1. List: [{NodeId:1 Addr:localhost:12345 Leader:true RaftIdx:4}]
====== APPLYING MEMBERSHIP UPDATE: {NodeId:1 Addr:localhost:12345 Leader:true RaftIdx:3}
----------------------------
Group: 1. List: [{NodeId:1 Addr:localhost:12345 Leader:true RaftIdx:4}]
Group: 0. List: [{NodeId:1 Addr:localhost:12345 Leader:true RaftIdx:3}]
2017/01/05 13:57:14 error.go:75: Indexing requires ICU to be enabled.
```

Then running "dgraphloader -r goldendata.rdf.gz" give 

```
Retrying req: 47. Error: rpc error: code = 14 desc = grpc: the connection is unavailable
Retrying req: 20. Error: rpc error: code = 14 desc = grpc: the conneRetrying req: 24. Error: rpc error: code = 14 desc = grpc: the connection is unavailable
Retrying req: 50. Error: rpc error: code = 14 desc = grpc: the connection is unavailable
Retrying req: 64. Error: rpc error: code = 14 desc = grpc: the connection is unavailable
Retrying req: 97. Error: rpc error: code = 14 desc = grpc: the connection is unavailable
Retrying req: 94. Error: rpc error: code = 14 desc = grpc: the connection is unavailable
Retrying req: 88. Error: rpc error: code = 14 desc = grpc: the connection is unavailable
Retrying req: 25. Error: rpc error: code = 14 desc = grpc: the connection is unavailable
Retrying req: 44. Error: rpc error: code = 14 desc = grpc: the connection is unavailable
Retrying req: 90. Error: rpc error: code = 14 desc = grpc: the connection is unavailable
```

**Comment by pawanrawal** [comment]
Hey @hkgrani 

I think the `ICU_DATA` variable is not set for you. You can verify this by running `echo $ICU_DATA`. The Step 1 of the installation mentions how to set it. 

You can do a `export ICU_DATA=/usr/local/share/icudt58l.dat` to set it. You can also make it part of your `~/.bashrc` or `~/.zshrc`.

**Comment by hkgrani** [comment]
Thanks @pawanrawal . Yes the variable was not set (or picked up) properly. I just "blindly" followed the instructions and the export was put in a newly created .bash file. It seems that that variable was not picked up by my terminal window (I am using iTerm). When I copied export over to .bash_profile where I keep the other environment variables it was picked up and I was able to start the server, import the data and do the queries in the 5. step tutorial.

Thanks for the help on this. Let me know if keeping the variable in .bash_profile will cause any other issues later, if not i will keep it there. Issue can be closed
